rplugin/python3/ultest/handler/parsers/output.py

```diff
@@ -40,6 +40,9 @@ class OutputPatterns:
         namespace_separator=" › ",
     ),
     "elixir#exunit": OutputPatterns(failed_test=r"\s*\d\) test (?P<name>.*) \(.*\)$"),
+    "php#phpunit": OutputPatterns(
+        failed_test=r"\s*\d\)(?P<namespace>.*)::(?P<name>.*)",
```

Comment:
I believe what you want is
```python
        failed_test=r"\s*\d\) (?P<namespaces>.*)::(?P<name>.*)",
        namespace_separator=r"\\"

```
which will give you two namespaces "Tests" and "FakeTest", would that be correct? If you have the test file that'd be great as well

Reply:
Thanks for the tips. The testfile is quite simple:

```php
<?php

namespace Tests;

class FakeTest extends TestCase
{
    public function test_fake()
    {
        $this->assertEquals(0.0, 1.0);
    }

    public function test_fake_2()
    {
        $this->assertFalse(false);
    }
}
```

I'm not sure if it's correct that it matches two namespaces. As `Tests` is the namespace and  `FakeTest` the class name.